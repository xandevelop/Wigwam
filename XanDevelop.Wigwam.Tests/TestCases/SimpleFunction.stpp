test | as a user I can say hello
say hello

func | say hello
echo | hello