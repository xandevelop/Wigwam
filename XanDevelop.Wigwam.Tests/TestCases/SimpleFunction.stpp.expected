Start Test as a user I can say hello
    Start Func Call say hello
    End Func Call say hello
End Test as a user I can say hello

Start Function say hello
    Command | command=echo | target=hello | value=
End Function say hello